crates/next-api/src/nft_json.rs

```diff
@@ -133,6 +256,14 @@ impl Asset for NftJsonAsset {
                 continue;
             }
 
+            if let Some(ref exclude_globs) = exclude_globs
+                && exclude_globs
+                    .iter()
+                    .any(|glob| glob.matches(&referenced_chunk_path.path.clone()))
```

Comment:
this clone should be unnecessary

reference_chgunk_path.path.as_str() should work